src/components/sbb-header/sbb-header.scss

```diff
@@ -34,8 +33,13 @@
   --sbb-header-inset-block-start: calc(var(--sbb-header-height) * -1);
 }
 
+:host([hide-on-scroll][data-fixed][data-animated]) {
```

Comment:
Nitpick; It is not immediatelly clear to me _when_ this is applied? Could this maybe be documented?